### PR TITLE
[HELIUM-275] Notifications

### PR DESCRIPTION
**Pull request by allenan**
there are still two things left to do with this:
- [x] fix a bug in the paginated notifications table which occasionally causes active notifications to be displayed as inactive
- [ ] implement "seen by x teammates". Right now it's just a UI stub

### CODE REVIEW

File: assets/js/components/notifications/Notifications.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+class Notifications extends Component {
+  constructor(props) {
+    super(props)
+    this.markRead = this.markRead.bind(this)
+  }
+
+  markRead(notification, followUrl = true) {
+    const { markRead } = this.props
+    if (followUrl) {
+      markRead(notification.id, {active: false}, notification.url)
```
Comment (vicmgs) [inline assets/js/components/notifications/Notifications.jsx]: are these params used? active: false

Comment (allenan) [reply assets/js/components/notifications/Notifications.jsx]: oh yeah, not anymore. originally it was an updateNotification query before I changed around the data structure

File: lib/console_web/controllers/notification_controller.ex [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  alias Console.Notifications.Notification
+
+  action_fallback(ConsoleWeb.FallbackController)
+
+  def create(conn, %{"notification" => params}) do
+    current_team = conn.assigns.current_team
+
+    with {:ok, notification = %Notification{}} <- Notifications.create_notification(current_team, params) do
+      broadcast(notification, "new")
+      # AuditTrails.create_audit_trail("device", "create", current_user, current_team, "devices", device)
```
Comment (vicmgs) [inline lib/console_web/controllers/notification_controller.ex]: copy paste leftover?

File: lib/console_web/schema/schema.ex [hunk 3]
```diff
@@ -134,7 +144,13 @@ defmodule ConsoleWeb.Schema do
     @desc "Get paginated audit trails"
     paginated field :audit_trails, :paginated_audit_trails do
       arg :user_id, :string
-      resolve(&Console.Devices.AuditResolver.paginate/2)
+      resolve(&Console.AuditTrails.AuditResolver.paginate/2)
```
Comment (vicmgs) [inline lib/console_web/schema/schema.ex]: nice, one

**Comment by allenan** [comment]
thanks, i'll go ahead and merge this and implement "seen by" in a separate PR

**Pull request merged**